app/src/main/java/com/example/ar_reshare/SplashActivity.java

```diff
@@ -59,7 +59,7 @@ private void proceed() {
         timer.schedule(new TimerTask() {
             @Override
             public void run() {
-                Intent intent = new Intent(SplashActivity.this, ARActivity.class);
+                Intent intent = new Intent(SplashActivity.this, FallbackActivity.class);
```

Comment:
I do not think this should have been committed Lingtao! I am assuming you did it just for testing.

Reply:
Yeah, it is changed for testing due to the simulator's AR, I have now fixed it!